release_docs/RELEASE.txt

```diff
@@ -565,6 +565,12 @@ Bug Fixes since HDF5-2.0.0 release
 
     Configuration
     -------------
+    - The RPATH settings were restored by appending to the CMAKE_INSTALL_RPATH
+
+      The RPATH settings were removed by a pull-request #5271, but the settings are needed
+      under certain conditions. The settings have been restored by appending the necessary paths
+      and will not override and existing settings.
```

Comment:
and will not override and existing settings.

override or overwrite?
override any existing settings?

Reply:
Maybe document that relative rpaths ($ORIGIN / @loader_path) are always added (and that's it).